src/AssetsPage/AssetsTable/index.jsx

```diff
@@ -9,61 +9,94 @@ import { connect } from 'react-redux';
 
 import FontAwesomeStyles from 'font-awesome/css/font-awesome.min.css';
 import { assetActions } from '../../data/constants/actionTypes';
-import { clearAssetsStatus, deleteAsset } from '../../data/actions/assets';
+import { clearAssetsStatus, deleteAsset, sortUpdate } from '../../data/actions/assets';
 
 export class AssetsTable extends React.Component {
   constructor(props) {
     super(props);
     this.state = {
-      displayAssetsList: [],
       modalOpen: false,
       statusAlertOpen: false,
       assetToDelete: {},
+      deletedAsset: {},
+      deletedAssetIndex: null,
       elementToFocusOnModalClose: {},
-      statusAlertRef: {},
     };
 
+    this.columns = {
+      display_name: {
+        label: 'Name',
+        key: 'display_name',
+        columnSortable: true,
+      },
+      content_type: {
+        label: 'Type',
+        key: 'content_type',
+        columnSortable: true,
+      },
+      date_added: {
+        label: 'Date Added',
+        key: 'date_added',
+        columnSortable: true,
+      },
+      delete_asset: {
+        label: 'Delete Asset',
+        key: 'delete_asset',
+        columnSortable: false,
+      },
+    };
+
+    this.trashcanRefs = {};
+    this.statusAlertRef = {};
+
     this.onDeleteClick = this.onDeleteClick.bind(this);
     this.deleteAsset = this.deleteAsset.bind(this);
     this.addSupplementalTableElements = this.addSupplementalTableElements.bind(this);
     this.closeModal = this.closeModal.bind(this);
     this.closeStatusAlert = this.closeStatusAlert.bind(this);
     this.renderStatusAlert = this.renderStatusAlert.bind(this);
-
-    this.trashcanRefs = {};
   }
 
-  componentDidMount() {
```

Comment:
I removed componentDidMount in favor of computing the additional table elements on the fly in the render method, since we wanted to not keep two copies of essentially the same data in memory between the Redux store and the component state (in this.props.assetsList and this.state.displayAssetsList, respectively).